oshmem/mca/scoll/ucc/scoll_ucc_module.c

```diff
@@ -373,12 +444,12 @@ mca_scoll_ucc_comm_query(oshmem_group_t *osh_group, int *priority)
     OPAL_TIMING_ENV_INIT(comm_query);
 
     if (!cm->libucc_initialized) {
-        if (0 < cm->nr_modules) {
+        if (memheap_map && memheap_map->n_segments > 0) {
             if (OSHMEM_SUCCESS != mca_scoll_ucc_init_ctx(osh_group)) {
                 cm->ucc_enable = 0;
                 return NULL;
             }
-        }
+        } 
```

Comment:
minor - extra space after }

Reply:
fixed. also added a memset on line 334 of scoll_ucc_module.c as UCC API changed to require this.